cupy/fallback_mode/fallback.py

```diff
@@ -0,0 +1,107 @@
+"""
+Main fallback class.
+"""
+import types
+import cupy as cp
+import numpy as np
+
+
+from cupy.fallback_mode.utils import _call_cupy
+from cupy.fallback_mode.utils import _call_numpy
```

Comment:
Importing non-module symbols is prohibited due to the [contribution guide](https://docs-cupy.chainer.org/en/stable/contribution.html#coding-guidelines).

Reply:
Sorry about that! Will fix it.